server/bastionai_app/src/main.rs

```diff
@@ -1,6 +1,14 @@
+use env_logger::Env;
```

Comment:
Lot's of changes here but I feel like this is mostly building on top of the core server, just check everything fits nicely once bert is merged